Dockerfile

```
@@ -14,3 +14,15 @@ ENV FLASK_ENV=production
 COPY . /app/
 ENTRYPOINT poetry run gunicorn --bind 0.0.0.0:5000 todo_app.wsgi:app
 
+FROM base as test
+RUN apt-get update && apt-get install curl -y
+RUN apt-get update && apt-get install wget -y
+#Install chrome
+RUN ls -lrt /etc/apt
+RUN apt-get update && apt-get install -y gnupg2
```

Can simplify this, which will make the image build faster
```suggestion
RUN apt-get update && apt-get install curl wget gnupg2 -y
```